src/porepy/numerics/mixed_dim/coupler.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, solver, coupling=None):
 
 #------------------------------------------------------------------------------#
 
-    def ndof(self, gb):
+    def ndof(self, gb, data_key=None):
```

Comment:
It is not clear to me what is `data_key` and who access this? Is it the discretization who decides?